### DISCUSSION TITLE
Virtual Chassis | Add all interfaces under VC from members

### DISCUSSION THREAD

**Discussion by deku-m** [netbox-community/netbox#11563] [Ideas]
Hi,

I didnt want to open up a new feature request immediatly. 
As it might been asked before but didnt found it yet.

I wonder how the idea is op Virtual Chassis as now we create the chassis and add the members under it.
Looking at how cisco models their "stacking" you notice that they virtual create one switch with members under them and when asking show interfaces the numbering is like 1/0/1-1/0/2,  2/0/1, 2/0/2. While you cant acces the underlaying second or third switch.
because it reachable by one. Physically it still seperated switches ofcourse.

Isnt it an idea to add all interfaces combined under a tab in virtual chassis and let the members have their own interfaces like now?
Most beautiful thing would be that it wil also create the interfaces like for example Cisco does. But i can imagine this isnt possible.

Mock up:
![afbeelding](https://user-images.githubusercontent.com/37069737/213984611-19716956-4947-4461-8c6f-720f14044713.png)

**Comment by PVM-NL** [comment]
I do see value in this. Created ticket if you dont mind:
https://github.com/netbox-community/netbox/issues/11590

Lets see what the developers think of it.

**Comment by shepherdjay** [comment]
All interfaces that aren't marked "management only" are reachable via the master device. This has come up before in #7692

Personally how virtual chassis are handled is one of the few issues I have with netbox. The assumption based on the responses to  #7692 is that the "master" device should be where all the virtual chassis items are configured because this is how switch stacks operate in reality.

However I actually think thats sacrificing a lot of usability for what is at most an implementation detail of specific technologies and not a required implementation for the concept of a shared management plane.
